test/integration/app-document/test/rendering.js

```diff
@@ -67,6 +67,18 @@ export default function ({ app }, suiteName, render, fetch) {
         expect($('#render-page-enhance-app').text().includes(nonce)).toBe(true)
         expect($('#render-page-enhance-component').text().includes(nonce)).toBe(true)
       })
+
+      // This is a workaround to fix https://github.com/zeit/next.js/issues/5860
+      // TODO: remove this workaround when https://bugs.webkit.org/show_bug.cgi?id=187726 is fixed.
+      test('It adds a timestamp to link tags with preload attribute to invalidate the cache (DEV only)', async () => {
```

Comment:
Could you also add a test to `integration/production` to make sure we don't regress there in the future.